src/main/java/org/sundry/recipes/model/Category.java

```diff
@@ -8,6 +10,8 @@
 /**
  * Created by kon1299 on 2019-06-25
  */
+@Data
```

Comment:
I would be surprised if this works with Hibernate. Most JPA frameworks like beans, IE public no args, setter based injection. You might need to add some annotations to get it to use this correctly.

Reply:
It works and application runs successfully in my local. Category has no other method except get and set. There is no constructor as well.